src/Components/DirectionBox/DirectionBox.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import ArrowRightAltIcon from "@material-ui/icons/ArrowRightAlt";
+import SyncAltIcon from "@material-ui/icons/SyncAlt";
+
+import { languages } from "../../utils/constants.js";
+
+import "./DirectionBox.scss";
+
+const DirectionBox = ({ direction = "default" }) => {
+  const foreignWordLanguage = useSelector(
```

Comment:
I think it's better if we use redux some levels down so that the component can be used stateles in other screens. The best way to transfer the values I think is via the props.